### ISSUE TITLE
Replace 'split' with 'concat' function

### ISSUE THREAD

**Issue by beckyjackson** [ontodev/valve.py#33]
See discussion https://github.com/ontodev/valve/issues/4

```
concat(slot, slot, slot, ...)
```

... where a slot is a datatype, function, or string. For example:

```
concat(label, " & ", gates)
```

Validates that the value matches: `[label] & [gates]` (`label` and `gates` are datatypes)

You could also do:

```
concat(label, " & ", list(", " gate))
```

This `concat` function would also replace `CURIE`:
```
concat(in(prefix.prefix), ":", local_name)
```

(this is slightly different than what @jamesaoverton proposed, which just used the table-column name instead of the `in` function)

**Comment by jamesaoverton** [comment]
We could also consider allowing regular expressions to match the separators. It would be easy for people to mess up, but useful.

**Comment by beckyjackson** [comment]
Agreed! I guess this could be done now with datatypes, but if it's just for separators it may make more sense to just use it once in the function.

**Comment by jamesaoverton** [comment]
Closed by #38
